main.js

```diff
@@ -78,45 +93,276 @@ const Keyboard = {
+    currElement.style.borderRadius = '';
   },
 
   isUpperMode() {
-    const capsLock = document.querySelector('.capslock');
+    const capsLock = document.getElementById('CapsLock');
+    const shift = document.getElementById('Shift');
     capsLock.addEventListener('click', () => {
       if (this.capsLock) {
         this.capsLock = false;
       } else {
         this.capsLock = true;
       }
-      console.log(this.capsLock);
     });
+    document.addEventListener('keydown', (e) => {
+      if (e.key === 'CapsLock') {
+        if (this.capsLock) {
+          this.capsLock = false;
+        } else {
+          this.capsLock = true;
+        }
+      }
+    });
+    shift.addEventListener('mousedown', (e) => {
+      this.capsLock = true;
+      e.target.addEventListener('mouseup', () => {
+        this.capsLock = false;
+      });
+    });
+  },
+
+  isActiveShift() {
+    document.addEventListener('keydown', (e) => {
+      if (this.isFind(e.target.className, 'shift') || e.key === 'Shift') {
+        if (this.capsLock) {
+          this.capsLock = false;
+        } else {
+          this.capsLock = true;
+        }
+        const symbols = document.querySelectorAll('.row-1 .btn');
+        for (let i = 0; i < symbols.length; i = 1 + i) {
+          if (this.lang === 'rus') {
+            symbols[i].innerHTML = this.symbols.rus[i];
+            symbols[i].id = this.symbols.rus[i];
+          } else {
+            symbols[i].innerHTML = this.symbols.eng[i];
+            symbols[i].id = this.symbols.eng[i];
+          }
+        }
+        e.target.addEventListener('keyup', () => {
+          if (this.capsLock) {
+            this.capsLock = false;
+          } else {
+            this.capsLock = true;
+          }
+          for (let i = 0; i < symbols.length; i = 1 + i) {
+            if (this.lang === 'rus') {
+              symbols[i].innerHTML = this.alphabet.rus[0][i];
+              symbols[i].id = this.alphabet.rus[0][i];
+            } else {
+              symbols[i].innerHTML = this.alphabet.eng[0][i];
+              symbols[i].id = this.alphabet.eng[0][i];
+            }
+          }
+        });
+      }
+    });
+    document.addEventListener('mousedown', (e) => {
+      if (this.isFind(e.target.className, 'shift') || e.key === 'Shift') {
+        const symbols = document.querySelectorAll('.row-1 .btn');
+        for (let i = 0; i < symbols.length; i = 1 + i) {
+          if (this.lang === 'rus') {
+            symbols[i].innerHTML = this.symbols.rus[i];
+            if (!this.isFind(symbols[i].className, symbols[i].innerHTML)) {
+              symbols[i].className += ` ${symbols[i].innerHTML}`;
+            }
+          } else {
+            symbols[i].innerHTML = this.symbols.eng[i];
+            if (!this.isFind(symbols[i].className, symbols[i].innerHTML)) {
+              symbols[i].className += ` ${symbols[i].innerHTML}`;
+            }
+          }
+        }
+        e.target.addEventListener('mouseup', () => {
```

Comment:
You create a new event listener on each `mousedown`.
So, the old ones are still present and each mouse click will lead to creating more and more event listeners. That's not good.
Usage of event delegation pattern and single event listener could fix the problem